app/controllers/review_controller.rb

```diff
@@ -10,7 +10,7 @@ def submit_for_2i
       status = "submitted_for_2i"
 
       if @step_by_step_page.update(
-        review_requester: current_user.name,
+        review_requester: current_user.uid,
```

Comment:
Is it worth fixing this up to the `Set status to submitted_for_2i when form submitted` commit? I can't think of any value in retaining the history that you temporarily opted for `name` before refactoring to `uid`.
Probably worth describing in the commit why `uid` was chosen too.

Reply:
I'd hadn't finished working through Bruce's comments, when I pushed this change. It can't be fixed up until the migration is as well.